recorder.c

```diff
@@ -1742,14 +1740,19 @@ void recorder_dump_on_signal(int sig)
         return;
 
     struct sigaction action;
+
+    // already set?
```

Comment:
I capitalized this comment and committed after running my base tests. I will run the Jenkins tests next, but see no reason they should fail.